### PR TITLE
Zilla example with SASL/SCRAM Authentication

### PR DESCRIPTION
**Pull request by ankit-kumar-25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,25 +4,26 @@
...
-| [http.kafka.sync](http.kafka.sync) | Correlates requests and responses over separate Kafka topics |
-| [http.kafka.async](http.kafka.async) | Correlates requests and responses over separate Kafka topics, asynchronously |
-| [http.kafka.cache](http.kafka.cache) | Serves cached responses from a Kafka topic, detect when updated |
-| [http.kafka.oneway](http.kafka.oneway) | Sends messages to a Kafka topic, fire-and-forget |
-| [amqp.reflect](amqp.reflect) | Echoes messages published to the AMQP server, broadcasting to all receiving AMQP clients |
-| [mqtt.reflect](mqtt.reflect) | Echoes messages published to the MQTT server, broadcasting to all receiving MQTT clients |
-| [sse.kafka.fanout](sse.kafka.fanout) | Streams messages published to a Kafka topic, applying conflation based on log compaction |
-| [sse.proxy.jwt](sse.proxy.jwt) | Proxies messages delivered by the SSE server, enforcing streaming security constraints |
-| [ws.echo](ws.echo) | Echoes messages sent to the WebSocket server |
-| [ws.reflect](ws.reflect) | Echoes messages sent to the WebSocket server, broadcasting to all WebSocket clients |
```
Comment (jfallows) [inline README.md:25]: Good job adding the new example to the table.
Please revert the spacing changes, no need for vertical alignment in raw markdown to get vertical alignment in rendered output.

File: http.kafka.async.sasl.scram/README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
+# http.kafka.async
```
Comment (jfallows) [inline http.kafka.async.sasl.scram/README.md:1]: Name should match example.

File: http.kafka.async.sasl.scram/README.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+Listens on http port `8080` or https port `9090` and will correlate requests and responses over the `items-requests` and `items-responses` topics in Kafka, asynchronously.
```
Comment (jfallows) [inline http.kafka.async.sasl.scram/README.md:2]: Update description to include the main idea, that we use SASL SCRAM authentication to Kafka.

**Review by jfallows — CHANGES_REQUESTED** [review]
This async example is a bit complex, and distracts from the main idea about SASL SCRAM, let's use the `http.kafka.oneway` as a base instead, as that is probably the simplest one for Kafka.

File: README.md (hunk 1 above)
Comment (ankit-kumar-25) [reply README.md]: done.

File: http.kafka.async.sasl.scram/README.md (hunk 3 above)
Comment (ankit-kumar-25) [reply http.kafka.async.sasl.scram/README.md]: done

File: http.kafka.async.sasl.scram/README.md (hunk 2 above)
Comment (ankit-kumar-25) [reply http.kafka.async.sasl.scram/README.md]: done

File: http.kafka.sasl.scram/stack.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+networks:
+  net0:
+    driver: overlay
+
+services:
+
+  zilla:
+    image: "ghcr.io/aklivity/zilla:develop-SNAPSHOT"
```
Comment (jfallows) [inline http.kafka.sasl.scram/stack.yml:10]: Please update to `latest` docker image. 😄

**Review by jfallows — CHANGES_REQUESTED** [review]

File: http.kafka.sasl.scram/stack.yml (hunk 4 above)
Comment (ankit-kumar-25) [reply http.kafka.sasl.scram/stack.yml]: done

**Review by jfallows — APPROVED** [review]

**Pull request merged**